src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilationWithAnalyzers.cs

```diff
@@ -708,6 +708,67 @@ public Task<ImmutableArray<Diagnostic>> GetAllDiagnosticsAsync()
             }
         }
 
+        private async Task GenerateCompilationEventsAndPopulateEventsCacheAsync(AnalysisScope analysisScope, AnalyzerDriver driver, CancellationToken cancellationToken)
+        {
+#if SIMULATED_EVENT_QUEUE
+            await _analysisState.GenerateSimulatedCompilationEventsAsync(analysisScope, _compilation, _compilationData.GetOrCreateCachedSemanticModel, driver, cancellationToken).ConfigureAwait(false);
```

Comment:
I have kept the simulated event queue code around for quicker investigation in case some performance issues arise. I'll delete it after a sprint or two.